src/utils/tooltip.ts

```diff
@@ -110,6 +110,11 @@ export default {
 
     const $inner = document.createElement('div');
     $inner.setAttribute('class', 'rk-tooltip-inner');
+
+    if (binding.modifiers.inTraceTable) {
+      $inner.setAttribute('class', 'rk-tooltip-inner trace-table-tooltip-inner');
```

Comment:
Utils files need to be shared and cannot be specific to specific files，please use namespace